packages/builder/package.json

```diff
@@ -29,7 +29,7 @@
     "coverage": "nyc.js ../../node_modules/mocha/bin/mocha",
     "build": "tsc --build .",
     "clean": "rimraf builtin-tasks internal *.d.ts *.map *.js tsconfig.tsbuildinfo build",
-    "algob": "mkdir -p project && cd project && node ../build/internal/cli/cli.js"
+    "algob:test": "rm -rf project-test && mkdir project-test && cd project-test && node ../build/internal/cli/cli.js init"
```

Comment:
You added `init` and decided to force-remove the directory. I made it a general task so that you could run other tasks too. The old command was non-intrusive. I.e. you could run the same task any times you want. If you want to remove the directory you can use `rm -rf && yarn algob init .` -- This will be the same as you did here. And it will be searchable in terminal (`Ctrl+r`), I use a system terminal to do it.

Can you restore the old command and rename `algob:test` to `algob:init`?
We can have two commands if you want to use `init` a lot. I use console command history and I don't have this issue.

Reply:
OK, updated.